src/Form/LegacyConsumer/FieldView.php

```diff
@@ -27,19 +27,30 @@ class FieldView {
 	 * @return void
 	 */
 	public static function render( Node $field, $formId ) {
-		$type = $field->getType();
-		$fieldIdAttribute = give( UniqueIdAttributeGenerator::class )->getId( $formId, $field->getName() );
+		$type                          = $field->getType();
+		$fieldIdAttribute              = give( UniqueIdAttributeGenerator::class )->getId( $formId, $field->getName() );
+		$visibilityConditionsAttribute = ( new static() )->getVisibilityConditionAttribute( $field );
```

Comment:
Why don’t you make the method static and just call it `static::getVisibilityConditionAttribute( $field )`, similar to what is already happening with `FieldView::getTemplatePath`? Nice thinking with making it a method in the first place, it definitely helps clean up this method with all that's going on in it.

Reply:
fixed: https://github.com/impress-org/givewp/compare/69424e0f4396...f49f40aa5a00